pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -4,12 +4,14 @@
 , qtbase
 }:
 with pythonPackages;
```

Comment:
I am pretty sure this is not allowed in pythonPackages.

Reply:
I do not like it neither, fixed.